migration/migration_pkg_test.go

```diff
@@ -263,3 +268,208 @@ func TestMigrateTrieNodesFromBitsetToTrieKey(t *testing.T) {
 	require.Equal(t, felt.Zero, trieNode.Left.Felt())
 	require.Equal(t, felt.Zero, trieNode.Right.Felt())
 }
+
+func TestSchemaMetadata(t *testing.T) {
+	t.Run("failure", func(t *testing.T) {
+		mockCtrl := gomock.NewController(t)
+		t.Cleanup(mockCtrl.Finish)
+		t.Run("at version", func(t *testing.T) {
```

Comment:
We generally don't test for db errors. Are the new db mocks necessary to increase coverage? We can force-merge if high coverage is unattainable.

Reply:
Without mocks, it's impossible to cover requested cover in `migrations.SchemaMetadata: lines 137-138, line 143`

Reply:
> Without mocks, it's impossible to cover 

Can we call `SchemaMetadata` on an empty db to cover 138-139? A call to `SchemaMetadata` with only the `db.SchemaVersion` key present should cover 145.

Does that sound good? That should allow us to remove the mocks.